deployments/multus-daemonset-thick-plugin.yml

```diff
@@ -178,6 +186,21 @@ spec:
             - name: cni
               mountPath: /host/etc/cni/net.d
               mountPropagation: Bidirectional
+        - name: config-cni-socket-permissions
+            image: ghcr.io/k8snetworkplumbingwg/multus-cni:thick
+            command:
+              - "chmod"
+              - "0700"
+              - "/host/var/run/multus-cni"
+            resources:
+              requests:
+                cpu: "10m"
+                memory: "15Mi"
+            securityContext:
+              privileged: true
+            volumeMounts:
+              - name: host-var-run-multus-cni
+                mountPath: /host/var/run/multus-cni
```

Comment:
> - where to put the code that sets up the multus socket directory permissions
> - how we setup that directory

How about to make it in server side (not initContainer, just in multus-daemon)? In this case, we just provide a way to configure as command line option.

Reply:
And let me ask you another question. Why do you need to create the directory in initContainer?

Reply:
> And let me ask you another question. Why do you need to create the directory in initContainer?

The directory is bind mounted into the host. AFAIU, that is what creates the directory, not "me". 

Regarding having the server setup the permissions for the directory, I've replied to that already in https://github.com/k8snetworkplumbingwg/multus-cni/pull/771#discussion_r800837670; I think it's past it's responsibility. The server should consume an already existing - and configured - directory, and fail if the permissions are insecure. 

What are your reasons to avoid using this particular init container ?